packages/react-player/cypress/component/player-controls.cy.tsx

```diff
@@ -0,0 +1,203 @@
+/**
+ * Copyright 2023 Design Barn Inc.
+ */
+
+import { PlayerState } from 'common';
+import React from 'react';
+
+import { Controls } from '../../src/controls';
+import { DotLottiePlayer } from '../../src/react-player';
+import { PlayerStateWrapper } from '../support/player-state-wrapper';
+
+describe('Controls', () => {
+  it('should render controls.', () => {
+    cy.mount(
+      <PlayerStateWrapper>
+        <DotLottiePlayer
+          // eslint-disable-next-line no-secrets/no-secrets
+          src={`https://lottie.host/ffebcde0-ed6d-451a-b86a-35f693f249d7/7BMTlaBW7h.lottie`}
+          style={{ height: '400px', display: 'inline-block' }}
+          autoplay
+        >
+          <Controls />
+        </DotLottiePlayer>
+        ,
+      </PlayerStateWrapper>,
+    );
+    cy.get('[aria-label="lottie-animation-controls"]').should('exist');
+  });
+
+  it('should not render controls without `controls` prop', () => {
```

Comment:
The title says should not render, but you add the Controls component and and tests that it should exist?